src/commonMain/kotlin/uk/gov/hmrc/calculator/utils/clarification/Clarification.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.calculator.utils.clarification
+
+// Description added for readability clarifying what each Clarification enum represent, no real usage in code yet.
+enum class Clarification(private val description: String? = null) {
```

Comment:
All of the Clarifications have a description, so although it's not being used currently is there any value in allowing the description to be null? 
```suggestion
enum class Clarification(private val description: String) {
```